bot/exts/utilities/bookmark.py

```diff
@@ -25,6 +23,53 @@
 )
 
 
+class LinkTargetMessage(discord.ui.View):
+    """The button that relays the user to the bookmarked message."""
+
+    def __init__(
+        self,
+        target_message: discord.Message
+    ):
+        super().__init__()
+        self.add_item(discord.ui.Button(label="View Message", url=target_message.jump_url))
+
+
+class SendBookmark(discord.ui.Button):
+    """The button that sends the bookmark to other users."""
+
+    def __init__(
+        self,
+        bookmark_function: Callable[
+            [discord.TextChannel, discord.Member, discord.Message, str], None
+        ],
+        author: discord.Member,
+        channel: discord.TextChannel,
+        target_message: discord.Message,
+    ):
+
+        self.bookmark_function = bookmark_function
+        self.clicked: list[int] = [author.id]
+        self.channel = channel
+        self.target_message = target_message
+
+        super().__init__(label="Receive Bookmark", style=discord.ButtonStyle.green)
+
+    async def callback(self, interaction: discord.Interaction) -> None:
+        """The callback to the function."""
```

Comment:
Maybe to be more clear?
```suggestion
        """The button callback."""
```